deploy/kubei.yaml

```diff
@@ -202,11 +202,11 @@ spec:
             done
       containers:
       - name: kubei
-        image: gcr.io/eticloud/k8sec/kubei:1.0.11
+        image: gcr.io/eticloud/k8sec/kubei:1.0.12
```

Comment:
@akpsgit  why kubei need new tag?

Reply:
not really needed. this commit (nor the previous) related to user experience in kubei

Reply:
so we should not create new tag here, right?